talk/setup.tex

```diff
@@ -275,5 +275,7 @@
 }
 
 \hypersetup{
+  colorlinks=true,
+  allcolors=.,
   urlcolor={\color{blue!50!white}}
```

Comment:
Like this:
```suggestion
  urlcolor={blue!50!white}
```
The `\color` is prepended also by hyperref

Reply:
Thx for spotting this!